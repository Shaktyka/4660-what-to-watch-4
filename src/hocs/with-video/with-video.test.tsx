import * as React from 'react';
import * as renderer from 'react-test-renderer';

import withVideo from './with-video';

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withVideo(MockComponent);

describe(`withVideo rendering`, () => {

  it(`withVideo renders correctly`, () => {
    const tree = renderer.create((
      <MockComponentWrapped
        src={` `}
        poster={` `}
        isPlaying={false}
        muted
      />
    ), {
      createNodeMock() {
        return {};
      }
    }).toJSON();

    expect(tree).toMatchSnapshot();
  });

});
