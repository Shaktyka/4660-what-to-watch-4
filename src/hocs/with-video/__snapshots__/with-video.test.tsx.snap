// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withVideo rendering withVideo renders correctly 1`] = `
<div>
  <video
    data-type="video/webm"
    height={175}
    poster=" "
    width={280}
  >
    Sorry, your browser doesn't support embedded video
  </video>
</div>
`;
